.kitchen.yml

```diff
@@ -27,8 +27,6 @@ platforms:
       - freebsd::pkgng
   - name: freebsd-10.1
     run_list: freebsd::portsnap
-  - name: ubuntu-14.04
```

Comment:
This is just removing the duplicate you can see in master: https://github.com/chef-cookbooks/omnibus/blob/master/.kitchen.yml#L30-L37